app/jobs/refresh_search_job.rb

```diff
@@ -0,0 +1,8 @@
+class RefreshSearchJob < ApplicationJob
```

Comment:
This job is here just if we need it later, depending on how the daily update gets implemented.